cachito/workers/nexus.py

```diff
@@ -191,16 +191,18 @@ def get_ca_cert():
 
 
 def get_component_info_from_nexus(
-    repository, component_format, name, version, group=None, max_attempts=1
+    repository, component_format, name, version=None, group=None, max_attempts=1
 ):
     """
     Get the component information from a Nexus repository using Nexus' REST API.
 
+    Nexus accepts wildcards for some of the parameters in a search query (e.g. name, version).
+    If you do use wildcards, make sure they will only match one component.
+
     :param str repository: the name of the repository
     :param str component_format: the format of the component (e.g. npm)
-    :param str name: the name of the component
-    :param str version: the version of the dependency; a wildcard can be specified but it should
-        not match more than a single version
+    :param str name: the name of the component, if format is raw then this is the unique identifier
+    :param str version: the version of the dependency, should be specified iff format is not raw
```

Comment:
"iff" is sometimes used for "if and only if", see https://en.wikipedia.org/wiki/If_and_only_if

But it will probably just be confusing for most people, I'll change this to "if" and leave the second logical implication for the error message.